src/iOS/Xamarin.iOS/Samples/Geometry/BufferList/BufferList.cs

```diff
@@ -316,7 +316,7 @@ public override void LoadView()
             _bufferDistanceEntry.Layer.CornerRadius = 4;
             _bufferDistanceEntry.LeftView = new UIView(new CGRect(0, 0, 5, 20));
             _bufferDistanceEntry.LeftViewMode = UITextFieldViewMode.Always;
-            _bufferDistanceEntry.KeyboardType = UIKeyboardType.NumberPad;
```

Comment:
Why was this change made? If the reason is to support decimals, I suggest using `DecimalPad` instead.

Reply:
NumberPad doesnt have a return key. If I use numberpad or decimal pad I would have to add a done button to the UI for when the user is done typing.